src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.less

```diff
@@ -18,6 +18,24 @@
       padding: 0;
     }
   }
+  #dep-editor {
+    min-height: unset;
+  }
+  #dependency {
+    margin: 0px !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
Because there is another important in another component which it is inheriting do used important to ensure this particular property is used

Reply:
I will let this be for now, but it has to be corrected. Because using !important is not a good practice unless we are overriding the 3rd party styles.